polybar-scripts/info-t-tasklist-summary/README.md

```diff
@@ -0,0 +1,8 @@
+#Summize Task List by Category
+
+Requirments:
```

Comment:
Please take a closer look at the skeleton and format the `README.md` correctly. 
It can be found here: [skeleton](https://github.com/polybar/polybar-scripts/tree/master/skeleton)